conf/imperial.config

```diff
@@ -82,21 +85,21 @@ profiles {
 }
 
 executor {
-    $pbspro {
-        queueSize = 49
+    '$pbspro' {
```

Comment:
Is it ok to use single quotes? will the variable be replaced?

Reply:
I think if it's single quotes it won't be interpreted, so I think it should be OK

Reply:
@bentsherman can you confirm it's OK to put in quotes?

Reply:
should be fine with single quotes or no quotes

Reply:
Thank you!